serverless/src/util/urs/getAccessTokenFromJwtToken.js

```diff
@@ -0,0 +1,26 @@
+import { getDbConnection } from '../database/getDbConnection'
+import { getVerifiedJwtToken } from '../getVerifiedJwtToken'
+
+// Knex database connection object
+let dbConnection = null
+
+/**
+ * Returns the decrypted urs system credentials from Secrets Manager
+ */
+export const getAccessTokenFromJwtToken = async (jwtToken) => {
+  // Retrieve a connection to the database
+  dbConnection = await getDbConnection(dbConnection)
+
+  const { id } = getVerifiedJwtToken(jwtToken)
+
+  const existingUserTokens = await dbConnection('user_tokens')
```

Comment:
Consider returning `user_id` for use in methods that require both a user id and token from the jwtToken.